Create python-app.yml

HypnoFroggy opened the pull request:
по сути пайплайн, вроде рабочий и без тестов, потому что тестов нет

HypnoFroggy committed:
Create python-app.yml

по сути пайплайн, вроде рабочий и без тестов, потому что тестов нет

FooMaa commented:
Дак я там билда не особо увидел
Я вижу статический анализ кода. А где билд, может я и ошибаюсь, поправь, если ошибся))

HypnoFroggy committed:
Update python-app.yml

HypnoFroggy committed:
починил пайплан

HypnoFroggy committed:
Чиню пайплайн

изменения вроде должны дать контейнеру запускать qt приложения offscreen, а то он ругается во время запуска

MasterOfUniverses committed:
чиним пайплайн: отключили сборку, чиним ее

HypnoFroggy commented:
просто запустим в дев так, потом починим

MasterOfUniverses reviewed (approved):
сделали стат анализ, билд в новом реквесте
